src/app/components/colorpicker/index.html

```diff
@@ -57,23 +62,19 @@
       propertyName="initialColor"
       defaultValue="#FFF"
       isOptional="true">
-      Specifies an initial color to load in the colorpicker. This property is deprecated. As an alternative, we recommend the <stache-code>formControlName</stache-code> property on reactive forms or <stache-code>ngModel</stache-code> on template-driven forms. See the demo below for examples.
+      Specifies an initial color to load in the colorpicker. This property is deprecated. As an alternative, we recommend the <stache-code>formControlName</stache-code> property on reactive forms or <stache-code>ngModel</stache-code> on template-driven forms. See <a stacheRouterLink="." fragment="demo">the demo for examples</a>.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="outputFormat"
       defaultValue="rgba"
       isOptional="true">
-      Specifies the format to use for the value in a colorpicker when the colorpicker uses a native input element such as a standard text input or a button. This property accepts <stache-code>rgba</stache-code>, <stache-code>hex</stache-code>, or <stache-code>hsla</stache-code>, but we do not recommend using it because users never see or use its value. Instead, if you need to access this format value, see the demo below for an example of the recommended way to access the value.
+      Specifies the format to use for the color when the colorpicker uses a native input element such as a standard text input or a button. This property accepts <stache-code>rgba</stache-code>, <stache-code>hex</stache-code>, or <stache-code>hsla</stache-code>, but we do not recommend using it because users never see or use its value. Instead, if you need to access this format value, see <a stacheRouterLink="." fragment="demo">the demo for an example</a> of the recommended way to access the value.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="presetColors"
       defaultValue="['#333', '#888', '#EFEFEF', '#FFF']"
       isOptional="true">
-      Specifies an array of colors to load in the colorpicker as preset choices. The colorpicker displays the colors in a series of 12 boxes for users to select.
-    </sky-demo-page-property>
-    <sky-demo-page-property
-      propertyName="skyColorpickerInput">
-      Place this attribute on an input element or button element that is wrapped in a <stache-code>sky-colorpicker</stache-code> component to create the element and colorpicker. You must set this attribute to the instance of the <stache-code>sky-colorpicker</stache-code>. This directive uses <stache-code>NgModel</stache-code> to bind data.
+      Specifies an array of colors to load as preset choices. The colorpicker displays the colors in a series of 12 boxes for users to select.
     </sky-demo-page-property>
```

Comment:
We are missing a public `returnFormat` input: https://github.com/blackbaud/skyux-colorpicker/blob/master/src/app/public/modules/colorpicker/colorpicker-input.directive.ts#L92. Looking at the PRs in the repo, it looks like Trevor and Steve are the ones who worked on this component most recently. Can you check with them to see if we should add the `returnFormat` property to the docs?

Reply:
Confirmed with Trevor that the property has never been used and doesn't need documented.